src/navigation/waypoint.py

```diff
@@ -1,16 +1,17 @@
 from typing import List
 
 import numpy as np
+import rospy
 
 import tf2_ros
 from context import Context, Environment
 from drive import get_drive_command
 from aenum import Enum, NoAlias
 from state import BaseState
 
-STOP_THRESH = 0.5
-DRIVE_FWD_THRESH = 0.34  # 20 degrees
-NO_FIDUCIAL = -1
+STOP_THRESH = rospy.get_param("waypoint/stop_thresh", 0.5)
+DRIVE_FWD_THRESH = rospy.get_param("waypoint/drive_fwd_thresh", 0.34)  # 20 degrees
+NO_FIDUCIAL = rospy.get_param("waypoint/no_fiducial", -1)
```

Comment:
@qhdwight you should read this one on the other end too if you don't already

Reply:
what is NO_FIDUCIAL even being used for?

Reply:
yeah but I dont think we use it anywhere